hey_fireball.py

```diff
@@ -19,25 +19,33 @@
 BOT_ID = os.environ.get("BOT_ID")
 EMOJI = os.environ.get('EMOJI')
 POINTS = os.environ.get('POINTS')
-SELF_POINTS = os.environ.get('SELF_POINTS', "DISALLOW")
+SELF_POINTS = os.environ.get('SELF_POINTS', "ALLOW")
+NEG_POINTS = os.environ.get('NEG_POINTS', "ALLOW")
```

Comment:
I think the default should be DISALLOW.

Reply:
Agreed. I must have forgotten to switch those off when I was done testing